config/database.yml

```diff
@@ -16,10 +16,11 @@
 #
 default: &default
   adapter: postgresql
-  encoding: unicode
-  # For details on connection pooling, see Rails configuration guide
-  # https://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  timeout: 5000
+  host: localhost
```

Comment:
try to use env variables for confidentials details like username, password, api keys etc.

Reply:
We never push database.yml file on git, Put this file inside a gitignore, if we want to add this file for reference we can add with database.yml.example and don't add credentials inside that.